docs/configurations.md

```diff
@@ -0,0 +1,266 @@
+
+The Performance related configurations for Consensus Commit are as follows:
+
+| name                                                    | description                                                                    | default                                                           |
+|---------------------------------------------------------|--------------------------------------------------------------------------------|-------------------------------------------------------------------|
+| scalar.db.consensus_commit.parallel_executor_count      | The number of the executors (threads) for the parallel execution.              | 128                                                               |
+| scalar.db.consensus_commit.parallel_preparation.enabled | Whether or not the preparation phase is executed in parallel.                  | true                                                              |
+| scalar.db.consensus_commit.parallel_validation.enabled  | Whether or not the validation phase (in `EXTRA_READ`) is executed in parallel. | The value of `scalar.db.consensus_commit.parallel_commit.enabled` |
+| scalar.db.consensus_commit.parallel_commit.enabled      | Whether or not the commit phase is executed in parallel.                       | true                                                              |
+| scalar.db.consensus_commit.parallel_rollback.enabled    | Whether or not the rollback phase is executed in parallel.                     | The value of `scalar.db.consensus_commit.parallel_commit.enabled` |
+| scalar.db.consensus_commit.async_commit.enabled         | Whether or not the commit phase is executed asynchronously.                    | false                                                             |
+| scalar.db.consensus_commit.async_rollback.enabled       | Whether or not the rollback phase is executed asynchronously.                  | The value of `scalar.db.consensus_commit.async_commit.enabled`    |
+
+#### Underlying Storage/Database Configurations
+
+Consensus Commit has the storage abstraction layer and supports multiple underlying storages.
+You can specify the storage implementation by the `scalar.db.storage` property.
+
+The following describes the configurations available for each storage.
+
+- For Cassandra, the following configurations are available:
+
+| name                     | description                              | default |
+|--------------------------|------------------------------------------|---------|
+| scalar.db.storage        | `cassandra` should be specified.         | -       |
+| scalar.db.contact_points | Comma separated contact points.          |         |
+| scalar.db.contact_port   | Port number for all the contact points . |         |
+| scalar.db.username       | Username to access the database.         |         |
+| scalar.db.password       | Password to access the database.         |         |
+
+- For Cosmos DB, the following configurations are available:
```

Comment:
```suggestion
- For Cosmos DB for NoSQL, the following configurations are available:
```
It might be better to use a clear/concrete name since Cosmos DB means several kinds of databases. I think it is enough to describe the clear name once here.

What do you think?